techniques/system/distributePolicy/1.0/metadata.xml

```diff
@@ -50,15 +38,12 @@
 
   <SYSTEMVARS>
     <NAME>CLIENTSFOLDERS</NAME>
-    <NAME>SYSLOGPORT</NAME>
     <NAME>SHARED_FILES_FOLDER</NAME>
-    <NAME>STORE_ALL_CENTRALIZED_LOGS_IN_FILE</NAME>
```

Comment:
ha, this one is a regression - we should be able to use this value to store all log in all.log
maybe a configuration in rudder-relayd ?